web/client/plugins/widgetbuilder/CatalogServiceEditor.jsx

```diff
@@ -43,7 +43,7 @@ export default ({service: defaultService, catalogServices,
     const handleChangeServiceProperty = (property, value) => {
         setService({ ...service, [property]: value });
     };
-
+    // todo: add ifc to serviceTypes
```

Comment:
is this still needed? if not please remove it

Reply:
I added it as I thought adding ifc model can be used in widget builder for table widget specifically. 
But I think it is not needed now as the adding table in widget builder is for showing attribute table data.
@allyoucanmap